### PR TITLE
Update shop.js

### PR DESCRIPTION
**Pull request by inno14**

### CODE REVIEW

File: src/shop.js [hunk 1]
```diff
@@ -35,13 +35,13 @@ module.exports.items = [
...
   { id: "premium", name: "Premium Item Roulette", price: 25, currency: "coin", description: "Gives an higher chance on getting rare roles in item roulette", role: "606123666895274003", response: "You now have an higher chance on rare items in item roulette! Use ,pir to use the elite roulette." },
   { id: "elite", name: "Elite Item Roulette", price: 50, currency: "coin", description: "Gives the highest chance on getting rare roles in item roulette", role: "606123666257870868", response: "You now have the highest chance on rare items in item roulette! Use ,eir to use the elite roulette." },
-  //{ id: "lottery", name: "Lottery ticket", price: 5, currency: "coin", description: "Get a ticket to take part in the lottery!", role: "947629828771831888", response: "You can now enter <#947930500725616700> 🥳"}
+  { id: "auto", name: "Auto Reaction", price: 5, currency: "gems", description: "Get a auto reaction whenever someone pings you!", response: "Ask the staffs to get it at a appropriate place but before asking keep in mind that Carl bot must be in the server with emoji."}
```
Comment (Hackante) [inline src/shop.js:44]:
```suggestion
  { id: "auto", name: "Auto Reaction", price: 5, currency: "gem", description: "Get a auto reaction whenever someone pings you!", response: "Ask the staffs to get it at a appropriate place but before asking keep in mind that Carl bot must be in the server with emoji."}
```

(read line 21)

**Review by Hackante — CHANGES_REQUESTED** [review]

**Review by Hackante — COMMENTED** [review]
I will add some chnages too but they don't have anything to do with the new item

File: src/shop.js [hunk 2]
```diff
@@ -35,13 +34,13 @@ module.exports.items = [
...
   { id: "premium", name: "Premium Item Roulette", price: 25, currency: "coin", description: "Gives an higher chance on getting rare roles in item roulette", role: "606123666895274003", response: "You now have an higher chance on rare items in item roulette! Use ,pir to use the elite roulette." },
   { id: "elite", name: "Elite Item Roulette", price: 50, currency: "coin", description: "Gives the highest chance on getting rare roles in item roulette", role: "606123666257870868", response: "You now have the highest chance on rare items in item roulette! Use ,eir to use the elite roulette." },
-  //{ id: "lottery", name: "Lottery ticket", price: 5, currency: "coin", description: "Get a ticket to take part in the lottery!", role: "947629828771831888", response: "You can now enter <#947930500725616700> 🥳"}
+  { id: "auto", name: "Auto Reaction", price: 5, currency: "gem", description: "Get a auto reaction whenever someone pings you!", response: "Ask the staffs to get it at a appropriate place but before asking keep in mind that Carl bot must be in the server with emoji."}
```
Comment (Hackante) [inline src/shop.js:43]:
```suggestion
  { id: "auto", name: "Auto Reaction", price: 5, currency: "gem", description: "Get an auto reaction whenever someone pings you!", response: "Ask a staff member to get it at an appropriate place but before asking keep in mind that Carl bot must be in the server with the emoji."}
```

**Review by Hackante — CHANGES_REQUESTED** [review]
what do you mean by place? Do you mean time?

**Comment by 3061LRTAGSPKJMORMRT** [comment]
> what do you mean by place? Do you mean time?

@Hackante 

I think they mean channels. It might be weird if they are pinged in announcement channels so "place" probably refers to channel

File: src/shop.js [hunk 3]
```diff
@@ -35,13 +34,13 @@ module.exports.items = [
   { id: "special", name: "Special Color Role", price: 400, currency: "coin", description: "Get your own color role!" },
   { id: "grey", name: "Grey Color Role", price: 200, currency: "coin", description: "Gain the very rare Grey Color role", role: "606123657210757136" },
   //{ id: "ranked", name: "Ranked Pass", price: 250, currency: "coin", description: "Gain access to play ranked games!", role: "832845602009645116" },
-  //{ id: "emoji", name: "Emoji", price: 450, currency: "coin", description: "Gain access to the `+emoji` and `+emojilist` commands", role: "663389088354664477" },
+  { id: "emoji", name: "Emoji", price: 450, currency: "coin", description: "Gain access to `+write` command. Note the `+emoji` and `+emojilist ` are in progress and will be fixed asap.", role: "663389088354664477" },
```
Comment (Hackante) [inline src/shop.js:37]:
```suggestion
  { id: "emoji", name: "Emoji", price: 450, currency: "coin", description: "Gain access to `+write` command. Note: The `+emoji` and `+emojilist` commands are in progress and will be fixed as soon as possible.", role: "663389088354664477" },
```

File: src/shop.js [hunk 4]
```diff
@@ -35,13 +34,13 @@ module.exports.items = [
...
   { id: "premium", name: "Premium Item Roulette", price: 25, currency: "coin", description: "Gives an higher chance on getting rare roles in item roulette", role: "606123666895274003", response: "You now have an higher chance on rare items in item roulette! Use ,pir to use the elite roulette." },
   { id: "elite", name: "Elite Item Roulette", price: 50, currency: "coin", description: "Gives the highest chance on getting rare roles in item roulette", role: "606123666257870868", response: "You now have the highest chance on rare items in item roulette! Use ,eir to use the elite roulette." },
-  //{ id: "lottery", name: "Lottery ticket", price: 5, currency: "coin", description: "Get a ticket to take part in the lottery!", role: "947629828771831888", response: "You can now enter <#947930500725616700> 🥳"}
+  { id: "auto", name: "Auto Reaction", price: 5, currency: "gem", description: "Get an auto reaction whenever someone pings you!", response: "Ask a staff member to get it at an appropriate place but before asking keep in mind that Carl bot must be in the server with the emoji."}
```
Comment (Hackante) [inline src/shop.js:43]:
```suggestion
  { id: "auto", name: "Auto Reaction", price: 5, currency: "gem", description: "Get an auto reaction whenever someone pings you!", response: "Ask a staff member to get it at an appropriate place/channel. But before asking keep in mind that Carl-bot must be in the same server with the emoji."}
```

**Review by Hackante — CHANGES_REQUESTED** [review]
After that one I can merge it

File: src/shop.js (hunk 3 above)
Comment (Hackante) [inline src/shop.js]:
```suggestion
  { id: "emoji", name: "Emoji", price: 450, currency: "coin", description: "Gain access to `+write` command. Note: The `+emoji` and `+emojilist` commands are in progress and will be fixed as soon as possible.", role: "663389088354664477" },
```

File: src/shop.js (hunk 4 above)
Comment (Hackante) [inline src/shop.js]:
```suggestion
  { id: "auto", name: "Auto Reaction", price: 5, currency: "gem", description: "Get an auto reaction whenever someone pings you!", response: "Ask a staff member to get it at an appropriate place/channel. But before asking keep in mind that Carl-bot must be in the same server with the emoji."}
```

**Review by Hackante — CHANGES_REQUESTED** [review]
I wil commit them myself dw

**Comment by Hackante** [comment]
> I wil commit them myself dw

nvm I can't

**Comment by inno14** [comment]
> > I wil commit them myself dw
> 
> nvm I can't

Cuz i already did??

**Review by Hackante — APPROVED** [review]

**Pull request merged**